src/domain/UBGraphicsWidgetItem.cpp

```diff
@@ -510,6 +510,11 @@ void UBGraphicsWidgetItem::activeSceneChanged()
     registerAPI();
 }
 
+void UBGraphicsWidgetItem::setApiScene(std::shared_ptr<UBGraphicsScene>)
```

Comment:
scene seems to be unused, so if the purpose is just to call `registerAPI`, this `registerAPI` function should directly be called, isn't it ?

Reply:
Definitely, yes. I will change that.

Reply:
Removed unused `activeSceneChanged and renamed `setApiScene` to `initAPI` without parameters.